lib/v2/screens/authentication/recover/recover_account_found/interactor/viewmodels/current_remaining_time.dart

```diff
@@ -0,0 +1,16 @@
+class CurrentRemainingTime {
```

Comment:
this class might be reusable @RaulUrtecho

Reply:
Shoul I move now or later to.. domain shared i guess.

Reply:
Leave it for later.